en_pud-ud-test.conllu

```diff
@@ -16734,8 +16734,8 @@
 3	,	,	PUNCT	,	_	16	punct	16:punct	_
 4	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	5	nmod:poss	5:nmod:poss	_
 5	brother	brother	NOUN	NN	Number=Sing	16	nsubj:pass	16:nsubj:pass	_
-6	Prince	prince	NOUN	NN	Number=Sing	5	appos	5:appos	_
-7	Henry	Henry	PROPN	NNP	Number=Sing	6	flat	6:flat	SpaceAfter=No
+6	Prince	prince	NOUN	NN	Number=Sing	6	nmod:desc	6:nmod:desc	_
+7	Henry	Henry	PROPN	NNP	Number=Sing	5	appos	5:appos	SpaceAfter=No
 8	,	,	PUNCT	,	_	9	punct	9:punct	_
 9	Duke	Duke	PROPN	NNP	Number=Sing	5	appos	5:appos	_
```

Comment:
Probably more natural to attach this to "Prince Henry" i.e. `7:appos`

Reply:
Done